### ISSUE TITLE
Reacts > How do you react?

### ISSUE THREAD

**Issue by kaustubhhiware** [kaustubhhiware/facebook-archive#6]
- How often do you use love reacts?
- What friends' posts are you most likely to react to?
- Is their a friend for whose posts you use a single react?

**Comment by parimatrix** [comment]
Hi. This seems an interesting thing to start with. I would like to work on this. Could you please assign me this issue so that I can start working on it. Thanks !

**Comment by kaustubhhiware** [comment]
Sure!

Have a look at the data, and please share what you think you will be working upon, before starting to code.

This will save us time later.
I'm adding you as a collaborator, after which this issue can be assigned to you.

**Comment by parimatrix** [comment]
So, yeah! I had a look at the data. In the Likes part of the data, fb provides a JSON object of all my reactions on any post. For every post it shows the reaction (ie whether its a 'Like', 'wow', etc) and the timestamp and a string describing it.
eg: You liked Tony Kroos's post. :p 
The challenge would be extracting names from this string as there is no fixed format of the sentence. It varies from post to post.  So, we could try to do the following things:
1. Show the counts of different reactions.
2. Show the variation of number of reactions over different time periods.
3. Show the names of friends whose posts we reacted to the most, or top 5. Something like that.

Any feedback would be great. And yes, I accepted the invitation for the collaborator.
